tests/modules/teams/resources/test_options.py

```diff
@@ -0,0 +1,31 @@
+# pylint: disable=missing-docstring
+import pytest
+
+@pytest.mark.parametrize('path,status_code', (
+    ('/api/v1/teams/', 401),
+    ('/api/v1/users/1', 401),
+))
+def test_teams_options_unauthorized(path, status_code, flask_app_client):
+    response = flask_app_client.options(path)
+
+    assert response.status_code == status_code
+
+
+@pytest.mark.parametrize('path,expected_allowed_methods', (
+    ('/api/v1/teams/', {'GET', 'POST', 'OPTIONS'}),
+    ('/api/v1/teams/1', {'GET', 'OPTIONS', 'PATCH', 'DELETE'}),
+    ('/api/v1/teams/2', {'OPTIONS'}),
+))
+def test_teams_options_authorized(
+        path,
+        expected_allowed_methods,
+        flask_app_client,
+        regular_user,
+        team_for_regular_user,
+        team_for_nobody
+    ):
+    with flask_app_client.login(regular_user, auth_scopes=('teams:write', 'teams:read')):
+        response = flask_app_client.options(path)
+
+    assert response.status_code == 204
+    assert expected_allowed_methods == set(response.headers['Allow'].split(', '))
```

Comment:
I will drive you crazy, but I want you change this thing once (I hope last) time again. Just like with the line above, you write and expression on the left, and "const" on the right... So I would like to swap the `expected_allowed_methods` and the right side.